src/esp/sim/SimulatorConfiguration.h

```diff
@@ -35,6 +35,12 @@ struct SimulatorConfiguration {
    * @brief Whether or not to load the semantic mesh
    */
   bool loadSemanticMesh = true;
+  /**
+   * Force creation of a separate semantic scene graph, even when no semantic
+   * mesh is loaded for the stage. This is required to support playback of any
+   * replay that includes a semantic-only render asset instance.
+   */
+  bool forceSeparateSemanticSceneGraph = false;
```

Comment:
What is the impact of this separation when no replay is desired?

Reply:
If your stage has a semantic mesh (and it loads successfully), there's no impact.

Otherwise, the impact is that your stage's RGBD mesh only gets used for RGBD observations, instead of for all observations. This is probably never desirable, so my comment here and in the docstring just says to set it to false (unless configuring for gfx replay).

I realize this is pretty confusing. It's hopefully temporary. In the long term, I'd like to see us refactor RGBD and Semantic scene graphs. The existing code makes some rigid assumptions about stages and objects, and it's a little confusing (see Simulator.py Sensor.draw_observation). I'm trying to avoid the concepts of stages and objects in my gfx replay code (all things are just render asset instances).